vayesta/solver/coupling.py

```diff
@@ -346,6 +514,9 @@ def _get_delta_t_for_delta_tailor(fragment, fock):
             mo_energy = (moa, mob)
 
     eris = fragment._eris
+    if eris is None:
+        # Symmetry-derived fragments may not have eris stored
+        eris = fragment.base.get_eris_array(cluster.c_active)
```

Comment:
Does this mean we perform integral transformation for each O(N_k) fragment?
That's OK as an initial solution to explore the method, but we should leave a TODO or similar,
because this wouldn't scale well

Reply:
I just realized this is just for the delta tailoring, which isn't really that relevant (?)

Reply:
TODO added.